### PR TITLE
feat(TDI-41954): Snowflake Commit and Rollback

### PR DESCRIPTION
**Pull request by mbasiuk-talend**
**What is the current behavior?** (You can also link to an open issue here)
https://jira.talendforge.org/browse/TDI-41954
no component for tSnowflakeCommit / tSnowflakeRollback

**What is the new behavior?**
Add 2 new components

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [x] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

JUnits is going to be add very soon.

### CODE REVIEW

File: components/components-snowflake/components-snowflake-definition/src/main/java/org/talend/components/snowflake/SnowflakeConnectionProperties.java [hunk 1]
```diff
@@ -12,6 +12,12 @@
 // ============================================================================
 package org.talend.components.snowflake;
 
+import static org.talend.components.snowflake.SnowflakeDefinition.*;
+import static org.talend.daikon.properties.presentation.Widget.widget;
+import static org.talend.daikon.properties.property.PropertyFactory.*;
```
Comment (undx) [inline components/components-snowflake/components-snowflake-definition/src/main/java/org/talend/components/snowflake/SnowflakeConnectionProperties.java]: disable .* imports in your editor.

File: components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeStandaloneOperation.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        return null;
+    }
+
+    @Override
+    public Schema getEndpointSchema(RuntimeContainer runtimeContainer, String s) throws IOException {
+        return null;
+    }
+
+    @Override
+    public ValidationResult validate(RuntimeContainer container) {
```
[the remark this answers is not in the document]
Comment (mbasiuk-talend) [reply components/components-snowflake/components-snowflake-runtime/src/main/java/org/talend/components/snowflake/runtime/SnowflakeStandaloneOperation.java]: I thought the same way in the beginning, but then checked tSnowflakeCloseSourceOrSink and it only performs a single operation as these 2 components.

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/122/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/122/)

**Comment by build-talend-common** [comment]
Pull request test result: [failure](https://ci-common.datapwn.com/job/components_pull-request_master/123/) [:x:](https://ci-common.datapwn.com/job/components_pull-request_master/123/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/124/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/124/) 
[Sonar results](https://sonar.datapwn.com/overview?id=org.talend.components%3Acomponents%3Ambasiuk%2FTDI-41954_snowflake_commit_rollback)

**Pull request merged**